app/src/main/java/com/ftw/hometerview/ui/updatenickname/UpdateNicknameActivity.kt

```diff
@@ -0,0 +1,94 @@
+package com.ftw.hometerview.ui.updatenickname
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Parcelable
+import android.util.Log
+import android.view.View
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.repeatOnLifecycle
+import com.ftw.hometerview.R
+import com.ftw.hometerview.databinding.ActivityBuildingReviewBinding
+import com.ftw.hometerview.databinding.ActivityUpdateNickNameBinding
+import com.ftw.hometerview.ui.buildinglist.BuildingListActivity
+import com.ftw.hometerview.ui.main.MainActivity
+import com.ftw.hometerview.ui.main.MainViewModel
+import com.ftw.hometerview.ui.searchcompanyresult.SearchCompanyResultActivity
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.launch
+import kotlinx.parcelize.Parcelize
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class UpdateNicknameActivity : AppCompatActivity() {
+
+    companion object {
+        val UPDATE_NICKNAME_RESULT_KEY = "UPDATE_NICKNAME_RESULT_KEY"
+        fun newIntent(context: Context, nickname: String): Intent {
+            return Intent(context, UpdateNicknameActivity::class.java)
+                .putExtra(UPDATE_NICKNAME_RESULT_KEY, nickname)
+        }
+    }
```

Comment:
상수명에 `Result` 가 포함되어 있어서 argument 를 set 할 때는 어울리지 않아보여서, UPDATE_NICKNAME_ARGUMENT_KEY 같은 이름의 상수를 따로 두면 어때?

```suggestion
    companion object {
        val UPDATE_NICKNAME_RESULT_KEY = "UPDATE_NICKNAME_RESULT_KEY"
        private val UPDATE_NICKNAME_ARGUMENT_KEY = "UPDATE_NICKNAME_ARGUMENT_KEY"
        fun newIntent(context: Context, nickname: String): Intent {
            return Intent(context, UpdateNicknameActivity::class.java)
                .putExtra(UPDATE_NICKNAME_ARGUMENT_KEY, Argument(nickname))
        }
    }
```

Reply:
좋아용~